frontend/src/custom-theme.scss

```diff
@@ -0,0 +1,35 @@
+
```

Comment:
This needs to be changed / removed because the original solution is NOT using scss, no need to add SCSS for a single component

Reply:
File removed, change updated on "fixes to contact list functionality" commit